doc/source/configuration/cephadm.rst

```diff
@@ -239,6 +239,21 @@ for Cinder, Cinder backup, Glance, and Nova in Kolla Ansible.
          mgr: "profile rbd pool=images"
        state: present
 
+Ceph Commands
+~~~~~~~~~~~~~
+
+It is possible to run an arbitrary list of commands against the cluster after deployment
+by setting the ``ceph_commands`` variable. ``ceph_commands`` should be a list of commands
+to pass to ``cephadm shell -- ceph``. For example:
+
+.. code:: yaml
+
+   # A list of commands to pass to cephadm shell -- ceph. See stackhpc.cephadm.commands
+   # for format.
+   cephadm_commands:
+     - "fs new cephfs cephfs_metadata cephfs_data"
```

Comment:
I would argue that's a good example, changing placement should rather be done using cephadm role, not the commands role.

Reply:
That is the example from the docs in the collection. I was going to use this approach to do:

```
ceph config set mgr mgr/prometheus/server_addr 10.0.0.1
```

So it wasn't bound to all interfaces. Is that a better example?

Reply:
Yes, an mgr var is a good example :-) I meant the old one is NOT a good example.

Reply:
Thanks, makes sense to not encourage a bad pattern of usage.